performance/tests/stockCardSummaries.yml

```diff
@@ -0,0 +1,24 @@
+execution:
+  - concurrency: 1
+    hold-for: 3m
```

Comment:
why 3 minutes? I see that other tests in stock management have 2 minutes

Reply:
I wanted to extend the time a bit, as one request takes about 40 seconds